client/my-sites/site-monitoring/components/site-monitoring-pie-chart/style.scss

```
@@ -2,6 +2,7 @@
 	.pie-chart__chart-drawing {
 		height: 250px;
 		width: 250px;
+		max-width: none;
```

Another alternative would be max-width 100%.
Both things work well.

```suggestion
		max-width: 100%;
```